From 4fefa0353c74004dd8636c2110b4f25ad0fc5cb6 Mon Sep 17 00:00:00 2001
From: Xin Liu <quic_liuxin@quicinc.com>
Date: Tue, 29 Oct 2024 11:12:20 +0800
Subject: [PATCH 1/3] FROMLIST: dt-bindings: watchdog: Document Qualcomm
 QCS8300

Add devicetree binding for watchdog present on Qualcomm
QCS8300 SoC.

Change-Id: I86c0835a265a1faf43842d11552d3f2259b3f857
Signed-off-by: Xin Liu <quic_liuxin@quicinc.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Link: https://lore.kernel.org/all/20241029031222.1653123-2-quic_liuxin@quicinc.com/
Upstream-Status: Submitted [https://lore.kernel.org/all/20241029031222.1653123-2-quic_liuxin@quicinc.com/]
---
 Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml b/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
index 219009d37b52..7984f692e723 100644
--- a/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
+++ b/Documentation/devicetree/bindings/watchdog/qcom-wdt.yaml
@@ -24,6 +24,7 @@ properties:
               - qcom,apss-wdt-msm8994
               - qcom,apss-wdt-qcm2290
               - qcom,apss-wdt-qcs404
+              - qcom,apss-wdt-qcs8300
               - qcom,apss-wdt-qcs615
               - qcom,apss-wdt-sa8775p
               - qcom,apss-wdt-sc7180
-- 
2.46.0

